ci/testnet-manager.sh

```diff
@@ -483,9 +494,10 @@ deploy() {
           ${maybeStop:+-S} \
           ${maybeDelete:+-D} \
           --stake-internal-nodes 1000000000000 \
-          --external-accounts-file /tmp/stakes.yml \
+          --external-accounts-file "$EXTERNAL_ACCOUNTS_FILE" \
           --lamports 8589934592000000000 \
-          --skip-deploy-update
+          --skip-deploy-update \
```

Comment:
As a follow-up PR if you'd prefer, we should be able to remove `--skip-deploy-update` now